### DISCUSSION TITLE
Any way to circumvent widevines encryption?

### DISCUSSION THREAD

**Discussion by arturfast** [seanpm2001/Its-time-to-cut-WideVine-DRM#15] [General]
Throughout the readme file, it is states that Widevine is insecure and can be broken. At some point it even claims L1 can be broken. I wonder how?

**Comment by xbjfk** [comment]
> Throughout the readme file, it is states that Widevine is insecure and can be broken. At some point it even claims L1 can be broken. I wonder how?

The methods are mostly kept confidential by piracy scene groups. Typically they involve hardware exploits, such as ARM TrustZone exploits. They use these to release Widevine L1 content, such as Netflix 4K content.
